### PR TITLE
Drop 4.0, hello 4.2!

### PR DESCRIPTION
**Pull request by pieterlexis**

### CODE REVIEW

File: vars/main.yml [hunk 1]
```diff
@@ -9,15 +9,6 @@ pdns_auth_powerdns_repo_master:
   yum_debug_symbols_repo_baseurl: "http://repo.powerdns.com/centos/$basearch/$releasever/auth-master/debug"
   name: "powerdns-auth-master"
 
-pdns_auth_powerdns_repo_40:
```
Comment (atosatto) [inline vars/main.yml]: I would keep this there for the people still using the 4.0 that have not migrated to 4.2.

Comment (pieterlexis) [reply vars/main.yml]: 4.0 will be EOL once 4.2 comes out though :)

Comment (atosatto) [reply vars/main.yml]: Sure, but 4.2 is not out yet. Let's remove the 4.0 repo in a separated PR to be merged once 4.2. is out.

**Comment by olemathias** [comment]
Can we get this merged? 4.2 is out now

**Pull request merged**